### PR TITLE
APEXMALHAR-2134 Fix the NullPointerException, if the kafka partition has no leader broker

### PR DESCRIPTION
**Pull request by chaithu14**

### CODE REVIEW

**Comment by chaithu14** [comment]
@siyuanh Please review and merge.

File: contrib/src/main/java/com/datatorrent/contrib/kafka/SimpleKafkaConsumer.java [hunk 1]
```diff
@@ -508,6 +508,9 @@ private void monitorMetadata()
               continue;
             }
             Broker b = pm.leader();
+            if (b == null) {
+              continue;
```
Comment (siyuanh) [inline contrib/src/main/java/com/datatorrent/contrib/kafka/SimpleKafkaConsumer.java:513]: We should put a INFO log here.

Comment (chaithu14) [reply contrib/src/main/java/com/datatorrent/contrib/kafka/SimpleKafkaConsumer.java:513]: Added a log message and updated.

**Comment by tweise** [comment]
@chaithu14 can you rebase the PR so we get the unit test fix into the CI build.

**Pull request merged**